src/plugins/plot/src/plot/MCTPlotController.js

```diff
@@ -322,12 +322,17 @@ define([
             return;
         }
 
+        let xDisplayRange = this.$scope.xAxis.get('displayRange'),
+            yDisplayRange = this.$scope.yAxis.get('displayRange');
+
+        if (!xDisplayRange || !yDisplayRange) {
```

Comment:
Why are these values potentially undefined? I think we need some explanatory comments here.

Reply:
I added comments. there are times where the ranges are undefined (when no plot data is available), and in the past this would cause a bunch of errors, especially using a trackpad, where zoom could happen unintentionally.

Reply:
Got it, makes sense, thanks.